lib/fluent/plugin/out_influxdb.rb

```diff
@@ -20,9 +21,10 @@ def initialize
 
   def configure(conf)
     super
-    @influxdb = InfluxDB::Client.new @dbname, host: @host, 
-                                              port: @port, 
-                                              username: @user, 
+    @remove_tag_prefix = conf['remove_tag_prefix']
```

Comment:
> Are you referring to the gsub below expecting a '.'?

Yes. In HandleTagNameMixin, users need to specify 'prefix.' for removing prefix but your patch configuration is `prefix`.
We should follow popular approach so using HandleTagNameMixin is better.

> I'll update this branch in a bit.

Thanks! After that, I will merge it.